README.rst

```diff
@@ -49,6 +49,30 @@ Running a simple example running a function every hour...
         except KeyboardInterrupt:
             print("Bye.")
 
+
+Specifying a timezone.
+----------------------
+
+For python 3.9+ you can use the standard library's ``zoneinfo`` module to specify a timezone.
+
+.. code:: python
+
+    import zoneinfo
+
+    async with Scheduler(tz=zoneinfo.ZoneInfo("Europe/Stockholm")) as scheduler:
+        ...
+
+
+
+For earlier python versions you can use a third party library like ``pytz``.
+
+.. code:: python
+
+    import pytz
+
+    async with Scheduler(tz=pytz.timezone("Europe/Stockholm")) as scheduler:
```

Comment:
Did you read this btw? https://lwn.net/Articles/870478/

goodbye stockholm timezone :D

Reply:
Ouch no I didn't know, best to use Europe/Berlin then?